### ISSUE TITLE
More information: Expend / Collapse

### ISSUE THREAD

**Issue by shawnthompson** [cenw-wscoe/WCAG-in-Plain-Language#37]
First off, I think this project is great and I'd love to put it on a11y.canada.ca!

As a signed user who uses a screen reader to help retain information I read, I've come to realize expend and collapse / detail summary doesn't work in reader view and you have to click to read the contents of it.

Do we really want to hide the content? I know it's a table, should it be a table (maybe another issue should be opened for this question)?

I spoke to others who use immersive readers and detail / summary are problematic.

Maybe adding a [Toggle](https://wet-boew.github.io/v4.0-ci/unmin/demos/toggle/toggle-en.html) button at the top of the table to allow users the choice to see everything might be a good option.

**Comment by andrewnordlund** [comment]
What I'd like to do is:
1) Put all information in a JSon file;
2) Write some Javascript to present the information in a different way; perhaps Cards instead of a table?

That's kinda what I did with the Nordburg Firefox extensions:  (https://www.nordburg.ca/ext/home.php vs https://www.nordburg.ca/ext/home.php?wet=false)  Same information: two presentation models.

Ohhhh, the Toggle thingie could work too.

**Comment by MaximPerry** [comment]
I like the idea. I've also recently learned how to read.json files with JS / JQuery, so it's on my to-do list. However, that .json file will still have to be manually updated (hardcoded) because you'd need something like PHP to write to files. GitHub Pages can't do that. 

However, we _could_ host this on a private domain, but this is now Government of Canada proprety. We would need to be granted permission to do so.

**Comment by shawnthompson** [comment]
Check this out:

It's a node project that converts an excel spreadsheet to a JSON file and then WET to filter.

https://a11y.canada.ca/wcag-issues/en/

https://github.com/gc-da11yn/wcag-issues
